src/SignalR/common/Http.Connections/src/Internal/HttpConnectionManager.cs

```diff
@@ -190,7 +198,7 @@ public void CloseConnections()
                 tasks.Add(DisposeAndRemoveAsync(c.Value.Connection, closeGracefully: false));
             }
 
-            Task.WaitAll(tasks.ToArray(), TimeSpan.FromSeconds(5));
+            await Task.WhenAll(tasks).NoThrow();
```

Comment:
Why is okay fire and forget when we were blocking ApplicationStopping cancellation on these tasks before?